src/features/CustomViews.ts

```diff
@@ -179,42 +176,57 @@ class HtmlContentView extends CustomView {
     }
 
     public getContent(): string {
-        let styleSrc = "none";
         let styleTags = "";
-
-        function getNonce(): number {
-            return Math.floor(Math.random() * 100000) + 100000;
-        }
-
         if (this.htmlContent.styleSheetPaths &&
             this.htmlContent.styleSheetPaths.length > 0) {
-            styleSrc = "";
             this.htmlContent.styleSheetPaths.forEach(
                 (p) => {
```

Comment:
Possible to change `p` to be more descriptive (to `path` maybe)? Is that the path to a css file?

Reply:
made it p so it didn't conflict with `import path from 'path';`... I'll change it to `styleSheetPath` or something